### PR TITLE
LoginFile adaptions

### PR DESCRIPTION
**Pull request by sisou**
4-times the pixels (double-sized, to look better on high-density screen such as phones), better wording and spacing.

### CODE REVIEW

File: src/lib/LoginFile.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ class LoginFile {
 
     static calculateQrPosition() {
         return {
-            x: 69,
-            y: 287,
+            x: 138,
+            y: 536,
```
Comment (danimoh) [inline src/lib/LoginFile.js]: So, this is not `2 * 287` on purpose?

File: src/lib/LoginFile.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class LoginFile {
     }
 
     filename() {
-        const walletName = 'Nimiq Login File '
+        const walletName = 'Nimiq LoginFile '
```
Comment (danimoh) [inline src/lib/LoginFile.js]: Needs to be changed to `Login File` now again.

File: src/lib/LoginFile.js [hunk 3]
```diff
@@ -115,11 +115,11 @@ class LoginFile {
         /* eslint-disable no-multi-spaces */
         const $canvas = QrEncoder.render({
             text: encodedSecret,
-            radius: 0.7,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
+            radius: 0.5,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
```
Comment (danimoh) [inline src/lib/LoginFile.js]: Comment still says `.7`.
That sentence and this line can just be removed. `.5` is already the default.

File: src/lib/LoginFile.js [hunk 4]
```diff
@@ -115,11 +115,11 @@ class LoginFile {
...
             ecLevel: 'M',   // the error-correction level to M. Thus we reduce the radius from .8 to .7
             fill: 'white',  // to reduce scanning issues.
             background: 'transparent',
-            size: Math.min(240, (window.innerWidth - 64)),
+            size: LoginFile.QR_SIZE,
         });
         /* eslint-enable no-multi-spaces */
         if (!$canvas) throw new Error('Cannot draw QR code');
```
Comment (danimoh) [inline src/lib/LoginFile.js]: Why does that canvas get appended to a div?

File: src/lib/LoginFile.js [hunk 5]
```diff
@@ -78,17 +78,17 @@ class LoginFile {
...
         await loaded;
-        this._ctx.drawImage(img, 58, 43, 198.6, 24);
+        this._ctx.drawImage(img, 116, 86, 397.2, 48);
```
Comment (danimoh) [inline src/lib/LoginFile.js]: Could use `this._drawDataUrlImage` here too?

File: src/lib/LoginFile.js (hunk 1 above)
Comment (sisou) [reply src/lib/LoginFile.js]: Every element except the logo and the date moved up, because of spacing changes.

File: src/lib/LoginFile.js (hunk 5 above)
Comment (sisou) [reply src/lib/LoginFile.js]: Oh, must have overlooked this. Thanks.

File: src/lib/LoginFile.js (hunk 4 above)
Comment (sisou) [reply src/lib/LoginFile.js]: It doesn't, apparently, but I did not change that in this PR :stuck_out_tongue: 
Removed it now ;)

File: src/lib/LoginFile.js [hunk 6]
```diff
@@ -102,28 +99,24 @@ class LoginFile {
 
     _drawWarningText() {
         const x = LoginFile.WIDTH / 2;
-        const y = LoginFile.HEIGHT - 32;
+        const y = LoginFile.HEIGHT - 86 - 2;
```
Comment (nibhar) [inline src/lib/LoginFile.js]: If not doing `LoginFile.HEIGHT - 88` I would prefer a comment explaining the different parts of the calculation.

File: src/lib/LoginFile.js [hunk 7]
```diff
@@ -102,28 +99,24 @@ class LoginFile {
...
      */
     _drawQrCode(encodedSecret) {
-        const $el = document.createElement('div');
-        /* eslint-disable no-multi-spaces */
         const $canvas = QrEncoder.render({
             text: encodedSecret,
-            radius: 0.7,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
-            ecLevel: 'M',   // the error-correction level to M. Thus we reduce the radius from .8 to .7
-            fill: 'white',  // to reduce scanning issues.
+            radius: 0.5,
```
Comment (nibhar) [inline src/lib/LoginFile.js]: Why did the radius decrease even more? Changing it from .8 to .7 was worth a comment. Why is it now .5 without the comment?

File: src/lib/LoginFile.js (hunk 6 above)
Comment (sisou) [reply src/lib/LoginFile.js]: Only because I don't want to write that number.

File: src/lib/LoginFile.js (hunk 7 above)
Comment (sisou) [reply src/lib/LoginFile.js]: Because it looks bad in such a high resolution. The gaps in the radiuses are very visible in high res.

Comment (danimoh) [reply src/lib/LoginFile.js]: Actually .5 is the maximum allowed value.
Values over .5 look glitchy.

Comment (sisou) [reply src/lib/LoginFile.js]: Yes. It's not too bad with low resolution, as the missing pixels hide it somewhat, but sadly it became obvious now.

**Pull request merged**